point_viewer_grpc/src/bin/octree_benchmark.rs

```diff
@@ -44,6 +44,10 @@ fn main() {
                 .help("Number of points to stream. [50000000]")
                 .long("num-points")
                 .takes_value(true),
+            clap::Arg::with_name("num-threads")
+                .help("Number of threads, 4 by default")
```

Comment:
The default is num(cpus) - 1, not 4, right?

Reply:
yes, done!